frontend/tests/project-creation.spec.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { test, expect } from '@playwright/test';
+
+const nearPixelArea = 50; // Specifies how far away an element can be when using near
+
+test('test the full project creation workflow', async ({ page }) => {
+  const testProjectName = `test-project-name-${Math.trunc(
+    Math.random() * 1000
+  )}`;
+  const testProjectDescription = 'test-project-description';
+
+  await page.goto('http://localhost:4200/');
+
+  // Check whether you got redirected to the /auth page
+  await expect(page).toHaveURL('http://localhost:4200/auth');
+
+  await page.locator('button:text-matches("Login with .*", "g")').click();
+
+  // Wait for the response from the backend
+  await page.waitForResponse('http://localhost:8000/api/v1/authentication/');
```

Comment:
```suggestion
  await page.waitForResponse(environment.backend_url + '/authentication/');
```

The tests should read the `environment.ts` and use the specified backend URL. This allows us to tests in different environments (e.g. in a cluster or staging environment).
I'd suggest to create a separate environment OR configuration file and use it for the playwright tests. In general, the configurable options should also be configurable for the tests. 

When using a custom playwright environment, you can import it with: 
```
import { environment } from 'src/environments/environment.dev';
```